src/main/java/kernitus/plugin/OldCombatMechanics/module/ModulePlayerCollisions.java

```diff
@@ -55,6 +56,12 @@ public void onPlayerChangeWorld(PlayerChangedWorldEvent e){
         createOrUpdateTeam(e.getPlayer());
     }
 
+    @EventHandler
+    public void onPlayerQuit(PlayerQuitEvent e) {
+        PacketManager.getInstance().removeListener(collisionPacketListener, e.getPlayer());
```

Comment:
I don't think this is needed, [as the PacketManager should handle that for you.](https://github.com/kernitus/BukkitOldCombatMechanics/blob/59ecdc25d9d6e59a20d9872f4f1a7a82fade40c9/src/main/java/kernitus/plugin/OldCombatMechanics/utilities/packet/PacketManager.java#L144)

Reply:
yeah, i just wasn't sure, it is a bit safer as packet listeners might be invoked async

Reply:
It might make it a bit more explicit, but the other event listens for PlayerQuit too on MONITOR priority.

`removeListener` should be fine with being called twice, I don't know if it is nicer to explicitly do that here again though.